<launch>
  <!-- Set this NIC IP to the interface *facing the sensor*, or 0.0.0.0 -->
  <arg name="bind_ip"   default="0.0.0.0"/>
  <arg name="sensor_ip" default="192.168.126.125"/>
  <arg name="frame_id"  default="ft_link"/>
  <arg name="topic"     default="wrench"/>

  <!-- If your sensor outputs counts, put your per-axis scales here (N/Nm per count).
       If web UI is set to "engineering units", leave all at 1.0 -->
  <rosparam param="scales">[1,1,1,1,1,1]</rosparam>

  <node pkg="com_3d" type="netft_publisher.py" name="netft" output="screen">
    <param name="bind_ip" value="$(arg bind_ip)"/>
    <param name="sensor_ip" value="$(arg sensor_ip)"/>
    <param name="frame_id" value="$(arg frame_id)"/>
    <param name="topic" value="$(arg topic)"/>
    <param name="pub_rate_hz" value="0"/>           <!-- 0 = no throttle -->
    <param name="csv_path" value=""/>               <!-- set e.g. /home/user/ft.csv -->
    <param name="csv_forces_only" value="true"/>
    <param name="csv_flush_every" value="50"/>
    <param name="zero_samples" value="200"/>
  </node>
</launch>
